Healthy/Classes/Modules/Dashboard/DashboardViewController.swift

```diff
@@ -33,6 +33,54 @@ final class DashboardViewController: UIViewController {
 
         configureView()
     }
+
+    // MARK: - Function for header section creation
```

Comment:
If you want to describe what the function does, don't use **// MARK**, instead use documentation comment. You can do so by pressing `options + cmd + /`

<img width="500" alt="image" src="https://github.com/motoon-eg/healthy/assets/42544598/e7ee2911-8032-4b92-8584-f7dade14108c">


Those info will show when you get the definition of a function like this:

<img width="500" alt="image" src="https://github.com/motoon-eg/healthy/assets/42544598/c33e5696-b837-4237-90c5-db996b719767">

Also this type of comment will help you to generate documentation for your app using DocC.